tools/build_workshop_materials.py

```diff
@@ -9,6 +9,8 @@
 #   "sphobjinv",
 #   "mdformat",
 #   "mdformat_myst",
+#   "distributed",
+#   "bokeh",
```

Comment:
Nope not here (these are the script deps), you want to add them down on line ~190